packages/sdks/package.json

```diff
@@ -54,5 +54,6 @@
       "path": "git-cz"
     }
   },
-  "packageManager": "yarn@3.2.0"
+  "packageManager": "yarn@3.2.0",
+  "resolutions": {}
```

Comment:
I think you manually removed the 2 lines here, which is not enough. You need to run `yarn run remove-symlinks`: that runs `yarn` at the end which removes the resolutions from `yarn.lock`

Reply:
Oh I try to only run the add/remove symlinks command but perhaps made a mistake